src/app/api/run/[runId]/approve/route.ts

```diff
@@ -16,6 +16,7 @@ export const POST = async (req: NextRequest, { params }: { params: { runId: stri
     }
 
     const filePath = path.join(UPLOAD_DIR, runId)
+    /* v8 ignore start */
```

Comment:
I think you might eventually be able to remove this comment by doing something similar that @kswanson33 might be doing for the setup app. Mocking the results of a function (in this case the global fetch)